sequential/installing/README.md

```diff
@@ -9,11 +9,13 @@ real coding. We will need some software:
 - Ensure that you have [Make](https://en.wikipedia.org/wiki/Make_(software))
   installed and running.
 
-Run `make` and if you get an output similar to this one, everything
-should be fine.
+### Checking environment
+
+1. Clone this repo `$> git clone https://github.com/lambdaclass/erlings.git`.
```

Comment:
this seems like too much text for such simple/common steps. Just show a single snippet with the code:

```
$> git clone https://github.com/lambdaclass/erlings.git
$> cd ~/erlings/sequential/installing/
$> make
```

"And you should get the following outpu:"